include/pfs/drp/stella/SpectraTemplates.hpp

```diff
@@ -503,37 +503,86 @@ bool pfs::drp::stella::Spectrum<SpectrumT, MaskT, VarianceT, WavelengthT>::ident
 
     ndarray::Array< double, 1, 1 > D_A1_FittedWLenCheck = math::getSubArray( D_A1_WLen, I_A1_IndexCheckPos );
 
+    std::vector<string> S_A1_Args(3);
```

Comment:
How do you know this code works? I don't see any changes to the test suite beyond those testing the narrow functionality of the polyfit routine itself. Can you add a test which demonstrates that it is being applied here properly?